Thing.h

```diff
@@ -461,7 +461,7 @@ class ThingEventObject {
       break;
     }
 
-    data["timestamp"] = timestamp;
+    //data["timestamp"] = timestamp;
```

Comment:
I think we can leave this, at least I don't see any immediate problems with having it.